day-one/upload-schema.mdx

```diff
@@ -1,5 +1,213 @@
 ---
-title: 'Upload Schema'
-description: 'Learn how enterprise developers can upload existing schemas to define data models in Amplication.'
-icon: 'database'
----
\ No newline at end of file
+title: "Upload Schema"
+description: "Use your existing database to jumpstart a new resource and generate Amplication entities and relationships by uploading its Prisma schema."
+sidebarTitle: "How to Upload Schema"
+---
+
+<Frame>
+    <img src="/images/day-one/upload-prisma-schema.png" />
+</Frame>
+
+If you have a database you'd like to use as the foundation for your new resource you can upload a `schema.prisma` file based on your database.
+Amplication will instantly create entities, fields, and relationships based on your schema, significantly accelerating your development process and letting you focus on business logic, not database setup.
+This feature quickly translates your existing database structure into Amplication's internal model, giving you a ready-to-go starting point.
+
+<Note>
+The **Upload Schema** feature is available when using the **.NET** or **Node.js** blueprints in Amplication.
+</Note>
+
```

Comment:
This feature is using The Prisma Introspection CLI. You can learn more about it here
https://www.prisma.io/docs/orm/prisma-schema/introspection